server/repositories/cardRepository.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('underscore');
+
+const logger = require('../log.js');
+const BaseRepository = require('./baseRepository.js');
+
+class DeckRepository extends BaseRepository {
+    getCards(brief, callback) {
```

Comment:
I might prefer named options, rather than changing the method name, though, mainly because they do similar things and it's nice to be able to default behaviour without having yet another method they both call